packages/databricks-vscode/src/extension.ts

```diff
@@ -847,19 +868,25 @@ export async function activate(
     const recordInitializationEvent = telemetry.start(
         Events.EXTENSION_INITIALIZATION
     );
-    bundleProjectManager
-        .configureWorkspace()
-        .catch((e) => {
-            recordInitializationEvent({success: false});
-            logging.NamedLogger.getOrCreate(Loggers.Extension).error(
-                "Failed to configure workspace",
-                e
-            );
-            window.showErrorMessage(e);
-        })
-        .finally(() => {
-            customWhenContext.setInitialized();
-        });
+
+    const configureWorkspace = () => {
+        bundleProjectManager
+            .configureWorkspace()
+            .catch((e) => {
+                recordInitializationEvent({success: false});
+                logging.NamedLogger.getOrCreate(Loggers.Extension).error(
+                    "Failed to configure workspace",
+                    e
+                );
+                window.showErrorMessage(e);
+            })
+            .finally(() => {
+                customWhenContext.setInitialized();
```

Comment:
We should reset the context to false when we start initialising in a new folder - to show `initializing...` welcome view

Reply:
Or it won't get shown anyway as we don't reset the tree view providers?

Reply:
Yeah. It will not show. Also, seems like we forget the target when we switch from a bad project (without databricks yaml for example), to a good one. Not fixing that edge case.